src/importer.js

```diff
@@ -0,0 +1,50 @@
+const fs = require('fs');
+const EventEmitter = require('events').EventEmitter;
+const csvjson = require('csvjson');
+
+const checkedFiles = [];
+
+class Importer extends EventEmitter {
```

Comment:
you don't use anything from `EventEmitter` here, so you don't need to extend from it.